web/html/src/manager/minion/ansible/accordion-path-content.tsx

```diff
@@ -0,0 +1,182 @@
+import * as React from "react";
+import { AnsiblePath } from "./ansible-path-type";
+import Network from "utils/network";
+import { Messages, Utils } from "components/messages";
+import { Loading } from "components/utils/Loading";
+import { AceEditor } from "components/ace-editor";
+
+type PropsType = {
+  path: AnsiblePath;
+};
+
+type StateType = {
+  open: boolean; 
```

Comment:
We use both `open` and `isOpen` mixed in different contexts for values like this so either works. I would recommend considering `isOpen` though since it's easier to grep for.